bin/runtests

```diff
@@ -49,6 +49,7 @@ fi
 DEPS_ADDONS=$(list_dependencies.py "$LOCAL_ADDONS")
 
 DB_NAME_TEST=${DB_NAME}_test
+sed -i "s/= ^${DB_NAME}/= ^${DB_NAME_TEST}/g" /etc/odoo.cfg
```

Comment:
As we use `--database $DB_NAME_TEST` in the command lines below in the script, I'd prefer to be consistent and use `--db-filter`